nixos/tests/home-assistant.nix

```diff
@@ -69,36 +69,36 @@ in {
   };
 
   testScript = ''
-    startAll;
-    $hass->waitForUnit("home-assistant.service");
+    start_all()
+    hass.wait_for_unit("home-assistant.service")
 
     # The config is specified using a Nix attribute set,
     # converted from JSON to YAML, and linked to the config dir
-    $hass->succeed("test -L ${configDir}/configuration.yaml");
+    hass.succeed("test -L ${configDir}/configuration.yaml")
     # The lovelace config is copied because lovelaceConfigWritable = true
-    $hass->succeed("test -f ${configDir}/ui-lovelace.yaml");
+    hass.succeed("test -f ${configDir}/ui-lovelace.yaml")
 
     # Check that Home Assistant's web interface and API can be reached
-    $hass->waitForOpenPort(8123);
```

Comment:
all comments could be the titles of a `subtest`, then this information would even be visible in the log later for more clarity.